i18n/fr/docusaurus-plugin-content-docs/current/5min-overview.md

```diff
@@ -150,7 +150,7 @@ Rencontrez le reste de l'équipe sur notre site web ou sur Linkedin : <https://w
 
 ### Chattez en votre langue
```

Comment:
Again not linked to this PR (sorry, I just highlight the little things I come across, continuous improvement ;)) : "Chattez *dans* votre langue".

Reply:
merci! fixed in https://github.com/alephium/wiki/pull/118/commits/cc648b062452f7d9350461a5addb712474bb4446